packages/@react-spectrum/label/test/StaticField.test.js

```diff
@@ -0,0 +1,273 @@
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(staticField).toBeInTheDocument();
+    expect(staticField).not.toHaveAttribute('aria-describedby');
+    expect(staticField).toHaveTextContent('2/3/2020 – 3/3/2020');
+  });
+
+  it('renders correctly with RangeValue<CalendarDateTime>', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        value={{start: new CalendarDateTime(2020, 2, 3, 12, 23, 24, 120), end: new CalendarDateTime(2020, 3, 3, 12, 23, 24, 120)}} />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(staticField).toBeInTheDocument();
+    expect(staticField).not.toHaveAttribute('aria-describedby');
+    expect(staticField).toHaveTextContent('2/3/2020 – 3/3/2020');
+  });
+
+  it('renders correctly with RangeValue<CalendarDate>', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        value={{start: new CalendarDate(2019, 6, 5), end: new CalendarDate(2019, 7, 5)}} />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(staticField).toBeInTheDocument();
+    expect(staticField).not.toHaveAttribute('aria-describedby');
+    expect(staticField).toHaveTextContent('6/5/2019 – 7/5/2019');
+  });
+
+  it('renders correctly with number value', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        value={10} />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(staticField).toBeInTheDocument();
+    expect(staticField).not.toHaveAttribute('aria-describedby');
+    expect(staticField).toHaveTextContent('10');
+    
+  });
+
+  it('renders correctly with RangeValue<NumberValue>', function () {
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        value={{start: 10, end: 20}} />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(staticField).toBeInTheDocument();
+    expect(staticField).not.toHaveAttribute('aria-describedby');
+    expect(staticField).toHaveTextContent('10 – 20');
+  });
+
+  it('attaches a user provided ref to the outer div', function () {
+    let ref = React.createRef();
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        ref={ref}
+        value="test" />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(ref.current).toBe(staticField);
+  });
+
+  it('attaches a user provided ref to the outer div with a label', function () {
+    let ref = React.createRef();
+    let {getByTestId} = render(
+      <StaticField
+        data-testid="test-id"
+        label="Field label"
+        ref={ref}
+        value="test" />
+    );
+
+    let staticField = getByTestId('test-id');
+    expect(ref.current).toBe(staticField);
```

Comment:
You may want to check that `ref.current` contains the StaticField label `span` or something to that effect just to make this test more robust.